environment

```diff
@@ -47,6 +47,7 @@ AWS_DEFAULT_OUTPUT=json
 AWS_DEFAULT_REGION=us-east-1
 GCP_DEFAULT_REGION=us-central1
 GCP_PROJECT_NAME=human-cell-atlas-travis-test
+DSS_AVAILABILITY_ZONES="${AWS_DEFAULT_REGION}a ${AWS_DEFAULT_REGION}c ${AWS_DEFAULT_REGION}d"
```

Comment:
This will fail for most regions that are not `us-east-1`.  Can this be written generically so that changing the region will allow it to work?

These also appear to be the zones used by allspark; any particular reason we don't use all zones?

Reply:
Availability zones can be introspected in TF scripting. This variable can be removed.